writings/2022-06-20-mev-boost-call-for-testing.mdx

```diff
@@ -0,0 +1,33 @@
+---
+slug: mev-boost-call-for-testing
+title: Boost your MEV in Ethereum proof-of-stake
+authors: [elopio]
+tags: [mev-boost]
+hide_table_of_contents: false
+---
+
+# Boost your MEV in Ethereum proof-of-stake
+
+Calling all validators to test `mev-boost` in the Kiln test network.
```

Comment:
```suggestion
Calling all validators to test [mev-boost](https://github.com/flashbots/mev-boost) in the Kiln and Ropsten test networks.
```

Reply:
I've added the link. About ropsten, let me se if I will have time to test both guides today.
Otherwise we can make a new call for testing with ropsten next week.